ktls/src/stream/error.rs

```diff
@@ -0,0 +1,88 @@
+//! Error type of `KtlsStream` and related operations.
+
+use std::{fmt, io};
+
+use rustls::{AlertDescription, InvalidMessage, PeerMisbehaved};
+
+#[non_exhaustive]
+#[derive(Debug)]
+/// The error type for `KtlsStream` and related operations.
+pub enum KtlsStreamError {
```

Comment:
This should really document that it is emitted within an `io::Error` from operations on `KtlsStream`.